HW_08/task3.py

```diff
@@ -0,0 +1,13 @@
+lst = ["qwe", "wEr", "erT", "rTY", "YuI"]
+
+# v1. via definition
+def in_upper(symbol):
+    return(symbol.upper())
```

Comment:
nit: brackets can be omitted 
```suggestion
    return symbol.upper()
```